src/store/reducers/ErrorReducer.js

```diff
@@ -8,6 +8,10 @@ const initialState = {
   registerError: {
     hasError: false,
     message: ''
+  },
+  notFoundError: {
+    hasError: false,
+    message: ''
   }
```

Comment:
nemoj da idemo ovako sa notFound, jednostavno pripemis state za movie i onda handlas 404 error tako sto odvedes ga na 404 stranicu, bez ovoga, jer not found je not found i handle-a se generalno tako sto odvedes korisnika na 404 starnicu